### ISSUE TITLE
Forbid unnamed services

### ISSUE THREAD

**Issue by DennisInSky** [gear-tech/sails#187]
### Problem to Solve

It is unclear how to address/get access to unnamed services. Another issue is that name can be considered as a part of the contract.

### Possible Solution

Let's forbid unnamed service for the time being

### Notes

It is blocked by #162 as all tests require abstraction of service whereas accessing unnamed services require abstraction of program
